ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -66,6 +66,75 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
         )
   }
 
+  @Explanation(
+    "This error happens when the combined use of all objects in a JVM heap pool exceed the configured limit."
+  )
+  @Resolution(
+    """The following actions can be taken:
+      |1. Review the historical use of heap space by inspecting the metric given in the message.
+      |2. Review the current heap space limits configured in the rate limiting configuration.
+      |3. Try to space out requests that are likely to require are large amount of memory to process."""
+  )
+  object HeapMemoryOverLimit
+      extends ErrorCode(
+        id = "HEAP_MEMORY_OVER_LIMIT",
+        ErrorCategory.ContentionOnSharedResources,
+      ) {
+    override def logLevel: Level = Level.INFO
+
+    case class Rejection(reason: String)(implicit errorLogger: ContextualizedErrorLogger)
+        extends DamlErrorWithDefiniteAnswer(
+          cause = s"The participant heap memory is over limit: $reason",
+          extraContext = Map("reason" -> reason),
+        )
+  }
+
+  @Explanation(
+    "This error happens when the number of concurrent gRPC streaming requests exceeds the configured limit."
+  )
+  @Resolution(
+    """The following actions can be taken:
+      |1. Review the historical need for concurrent streaming by inspecting the metric given in the message.
+      |2. Review the maximum streams limit configured in the rate limiting configuration.
+      |3. Try to space out streaming requests such that they do not need to run in parallel with each other."""
+  )
+  object MaximumNumberOfStreams
+      extends ErrorCode(
+        id = "MAXIMUM_NUMBER_OF_STREAMS",
+        ErrorCategory.ContentionOnSharedResources,
+      ) {
+    override def logLevel: Level = Level.INFO
```

Comment:
The loglevel for `ContentionOnSharedResources` is already INFO. Is it for the purpose of _"obviousness_ that this is overloaded here?